config/config.go

```diff
@@ -17,6 +17,7 @@ type Config struct {
 	GracefulShutdownTimeout    time.Duration `envconfig:"GRACEFUL_SHUTDOWN_TIMEOUT"`
 	HealthCheckInterval        time.Duration `envconfig:"HEALTHCHECK_INTERVAL"`
 	HealthCheckCriticalTimeout time.Duration `envconfig:"HEALTHCHECK_CRITICAL_TIMEOUT"`
+	ProfilesFeatureFlag        bool          `envconfig:"PROFILES_FEATURE_FLAG"`
```

Comment:
Can you rename this field/env var to something like `ENALBED_BREADCRUMBS` please?

The logic you've added is fine but the name I think is a bit too broad. Generally we want each feature flag to be specific to a particular feature/functionality. This allows us to toggle on/off some/none/all of the features. I know we are only using it for breadcrumbs at the moment but we could end up adding another flag and we want those to be independant of each other. As it is `ProfilesFeatureFlag` kind of encompasses any/all of the area profiles features. Sorry to be picky